server/routes/form.js

```diff
@@ -121,12 +121,19 @@ module.exports = function Index({
     '/categoriser/provisionalCategory/:bookingId',
     asyncMiddleware(async (req, res, transactionalDbClient) => {
       const section = 'categoriser'
-      const form = 'provisionalCategory'
+      const user = await userService.getUser(res.locals)
+      res.locals.user = { ...user, ...res.locals.user }
+      const isFemale = res.locals.user.activeCaseLoad.female
+      const form = isFemale ? 'womensProvisionalCategory' : 'provisionalCategory'
```

Comment:
Could we name it 'femaleProvisionalCategory' instead of 'womensProvisionalCategory'? Just to be consistent with the other elements introduced?

Reply:
done

Reply:
The corresponding HTML  page section is called womensProvisionalCategory so it is named this way to avoid causing confusion.